streams/lms_metadata.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python3
+"""LMS Metadata"""
+
+import argparse
+import re
+import fcntl
+import json
+import time
+import requests
+import subprocess
+from typing import Optional
+
+class LMSMetadataReader:
+  """A class for getting metadata from a Logitech Media Server."""
+
+  # meta_ref is probably an unneccessary variable to pass as an arg since it's obscured from the user, but we can eventually make it an optional setting for the user
+  def __init__(self, name: str, meta_ref: Optional[int] = 2, dump: Optional[bool] = False):
+    self.player_name = name
+    self.IP = None
```

Comment:
all caps would typically indicate a constant value. please make self.IP lowercase here

Reply:
I typically would name this ip_addr